### PR TITLE
first FriendList without mobx

### PR DESCRIPTION
**Pull request by JungEunPyo**
issue
#1

what are change
add in view/friend
adding new file(FriendListData, FriendMain, FriendMainFooter, FriendMainHeader)

FYI
nothing

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,12 +46,14 @@
...
     "eslint": "^7.0.0",
+    "faker": "^4.1.0",
     "jest": "^24.9.0",
     "metro-react-native-babel-preset": "^0.56.0",
     "patch-package": "^6.1.4",
     "react-native-size-matters": "^0.3.0",
     "react-native-svg": "^12.1.0",
-    "react-test-renderer": "16.9.0"
+    "react-test-renderer": "16.9.0",
+    "recyclerlistview": "^3.0.0"
```
Comment (sanghyunJ) [inline package.json]: 해당 라이브러리가 꼭 필요한 걸까요?
scrollview, sectionList flatList로 충분해 보입니다.

ScrollView vs FlatList - which one to use?

ScrollView renders all its react child components at once, but this has a performance downside.

Imagine you have a very long list of items you want to display, maybe several screens worth of content. Creating JS components and native views for everything all at once, much of which may not even be shown, will contribute to slow rendering and increased memory usage.

This is where FlatList comes into play. FlatList renders items lazily, when they are about to appear, and removes items that scroll way off screen to save memory and processing time.

https://reactnative.dev/docs/scrollview

react native doc을 보면 위와 같은 글이 있는데요. 한 번 읽어보시고, 친구리스트에는 어떤 컴포넌트가 어울릴지 선택해서 적용 부탁드려요.

**Comment by JungEunPyo** [comment]
수정해서 올리겠습니다~!!
-----Original Message-----

**Comment by JungEunPyo** [comment]
issue
#2

what are changed
change recycleview to scrollview
not using faker library

FYI
nothing

File: package.json (hunk 1 above)
Comment (sanghyunJ) [inline package.json]: 안쓰는 라이브러리는 yarn에서 삭제해주세요.

Comment (JungEunPyo) [reply package.json]: 넵~!!

**Comment by JungEunPyo** [comment]
issue
#5

what are changed
revmoe recycleview and faker from package.json file

FYI
nothing

**Comment by JungEunPyo** [comment]
issue
#5

what are changed
revmoe recycleview and faker from package.json file

FYI
nothing

**Comment by JungEunPyo** [comment]
issue
#3

what are changed
removed unusing library from package.json file

FYI
nothing

**Review by sanghyunJ — APPROVED** [review]
LGTM

**Comment by JungEunPyo** [comment]
issue
#4

what are changed

using URL to get friendListdata but it's not showing list when running application

FYI
nothing

**Comment by JungEunPyo** [comment]
issue
#4

what are changed

using URL to get friendListdata but it's not showing list when running application

FYI
nothing

**Pull request merged**